onnxruntime/core/framework/session_state.h

```diff
@@ -303,6 +304,10 @@ class SessionState {
   const InlinedHashSet<NodeIndex>* GetToBeExecutedRange(gsl::span<int const> fetch_mlvalue_idxs) const;
 #endif
 
+  std::unordered_map<std::string, std::unique_ptr<Tensor>>* GetMutableBufferedTensors() {
```

Comment:
Should return a reference instead. Because the returned value cannot be NULL.

Reply:
In the ProcessNode(), I want to differentiate the case that `name_to_buffered_tensor_` is passed in or not. By using pointer, the default value is nullptr. If the pointer is not nullptr, a map has been passed in explicitly.
If I use reference, the default value is an empty map object. In ProcessNode(), the reference always exist. I cannot tell that `name_to_buffered_tensor_` is passed in or not.